app/app.R

```diff
@@ -0,0 +1,62 @@
+library(shiny)
+library(DT)
+#create_metadata function
+
+# data 
+data <- readRDS("C:/Users/matthew.grainger/Documents/Projects_in_development/ECindicators/data/App_data.RDS")
```

Comment:
@DrMattG , I'm unable to test the app locally due to this

Reply:
Removed local path